source/utilities.tex

```diff
@@ -1357,18 +1427,17 @@
   template<class... Types>
     class tuple;
 
-  template<class... TTypes, class... UTypes, template<class> class TQual,
-           template<class> class UQual>
-    requires requires { typename tuple<common_reference_t<TQual<TTypes>, UQual<UTypes>>...>; }
-  struct basic_common_reference<tuple<TTypes...>, tuple<UTypes...>, TQual, UQual> {
-    using type = tuple<common_reference_t<TQual<TTypes>, UQual<UTypes>>...>;
-  };
+  template <typename T>
+    concept @\defexposconcept{tuple-like}@ = @\seebelownc@;         // \expos
```

Comment:
I'm confused - this *is* an exposition-only concept.  What am I missing?

Reply:
`\libconcept` and `\exposconcept` index uses of concepts. However, header pseudo-declarations of non exposition-only concepts don't use `\libconcept` and thus are not indexed. By using `\exposid` instead for exposition-only concepts, we achieve the same result.

Reply:
If you don't agree, this should at least be
```suggestion
    concept @\exposconcept{tuple-like}@ = @\seebelownc@;         // \expos
```
since it's not a definition.

Reply:
I do see precedence for using \exposconcept (see similar cases in memory.tex), so I went with that.  I'd suggest opening an edit issue to have these changed to \exposid if that's what you'd prefer.